src/ios/CDVWKInAppBrowser.m

```diff
@@ -913,16 +913,18 @@ - (void)createViews
         [self.toolbar setItems:@[self.closeButton, flexibleSpaceButton, self.backButton, fixedSpaceButton, self.forwardButton]];
     }
     
-    self.view.backgroundColor = [UIColor grayColor];
+    self.view.backgroundColor = IsAtLeastiOSVersion(@"13.0") ? [UIColor systemBackgroundColor] : [UIColor whiteColor];
     [self.view addSubview:self.toolbar];
     [self.view addSubview:self.addressLabel];
     [self.view addSubview:self.spinner];
-    
-    self.topBarView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, webViewBounds.size.width, webViewBounds.origin.y)];
-    self.topBarView.backgroundColor = [UIColor whiteColor];
-    [self.view addSubview:self.topBarView];
-    [self.view sendSubviewToBack:self.topBarView];
-    
+
+    UIEdgeInsets insets = [self getWindowSafeAreaInsets];
+    self.backgroundView = [[UIView alloc] initWithFrame:CGRectMake(0, insets.top, self.view.bounds.size.width, self.view.bounds.size.height - insets.top - insets.bottom)];
+    self.backgroundView.autoresizingMask = (UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight);
+    self.backgroundView.backgroundColor = [UIColor grayColor];
```

Comment:
What does this color affect and why is it hardcoded to gray?

Reply:
Same as above.